src/main/java/nl/knaw/dans/sword2/api/service/ServiceCollection.java

```diff
@@ -35,6 +38,16 @@ public class ServiceCollection {
     @XmlElement(namespace = "http://purl.org/net/sword/terms/")
     private boolean mediation;
 
+    private List<String> accept;
+
+    public List<String> getAcceptedMediaTypes() {
+        return accept;
+    }
+
+    public void SetAcceptedMediaTypes(String... mediaTypes) {
```

Comment:
```suggestion
    public void setAcceptedMediaTypes(String... mediaTypes) {
```

Use Java conventions, such as camelCasing method names.

Reply:
Thanks @janvanmansum 
- About sword:mediation, it is already implemented and works well too --> `<sword:mediation>false</sword:mediation>`
   See also new Service Document in Jira DD-1183
- About `app:accept@alternate=”multiparet-related”`, I thought I can explain that later in Jira issue or in a live conversation. I have checked this element in some other cases and examples. It seems not necessary to add an extra mention about it in Service Document. 
  -  One of the few examples in SWORD2 profile site:
     ```<workspace>
        <atom:title>Main Site</atom:title>
        <collection href="http://swordapp.org/col-IRI/43" >
            <atom:title>Collection 43</atom:title>

            <accept>application/xml</accept>
            <accept>application/zip</accept>
            <accept>application/atom+xml</accept>

            <sword:collectionPolicy>Collection Policy</sword:collectionPolicy>
            <dcterms:abstract>Collection description</dcterms:abstract>

            <sword:mediation>true</sword:mediation>
            <sword:treatment>treatment description</sword:treatment>
            <sword:packaging>http://purl.org/net/sword/package/BagIt</sword:packaging>
        </collection>
</workspace> ```
  - [AtomPub Multipart definition](https://datatracker.ietf.org/doc/html/draft-gregorio-atompub-multipart-04#section-5) says it is an extension to app:accept element, but we as DANS doesn't support it. 
   >  An AtomPub service announces that it will accept multipart/related
   POSTs by an extension to the app:accept element.  The "alternate"
   attribute's value MUST be one or more tokens, space-separated if more
   than one.  The only token defined by this specification is
   "multipart-related".  The presence of the "multipart-related" token
   in the 'alternate' attribute's value indicates that the collection
   accepts multipart/related POSTs whose Media Part has a content-type
   matching that specified in the app:accept element.  The following
   example indicates a collection that allows the creation of resources
   with the Ogg Bitstream Format and will also accept them in multipart
   form.
   `<app:accept alternate="multipart-related">application/ogg</app:accept>`
  - If we want to add `app:accept@alternate=”multiparet-related”` to the Service Document, it will be an empty element. It can be something as [IBM](https://www.ibm.com/docs/en/cics-ts/5.3?topic=feeds-creating-atom-service-document) suggest to write it:
  > Optional: If you do not want clients to create new Atom entries in one or more of your collections, include an empty <app:accept> element as a child element of the <app:collection> element.
For example:
```  <app:collection 
      href="http://www.example.com/customers/customercol.atom"> 
         <atom:title>Customer collection</atom:title>
         <app:accept/>
  </app:collection> 
```
- I'm going to commit solved typo problem of `Set/SetAcceptedMetaTypes`